### ISSUE TITLE
[BUG] Unable to get Pakistan Stock Exchange related stocks/ETFs

### ISSUE THREAD

**Issue by atiffarrukh** [ghostfolio/ghostfolio#3924]
**Bug Description**
Not sure if its a bug or configuration issue on my end.

Tried adding some stocks from KSE100 (PSX) for example, PSO.KAR but the app was not able to find it.
<img width="782" alt="image" src="https://github.com/user-attachments/assets/7896e06c-56bc-477d-9d45-ccbe2c407b01">

It can be done on hosted version (tried with free one)
<img width="792" alt="image" src="https://github.com/user-attachments/assets/2158d864-bd78-431d-afeb-e3eb00511918">

- Ghostfolio Version 2.114.0
- Self-hosted
- Browser: Chrome
- OS: MacOS

**Comment by dtslvr** [comment]
Hi @atiffarrukh

With [Ghostfolio Premium](https://ghostfol.io/en/pricing) you get access to professional data providers. As this is not considered a bug, I'll close this issue.

**Comment by atiffarrukh** [comment]
@dtslvr , its not available on selfhosted also?

**Comment by dtslvr** [comment]
> its not available on selfhosted also?

I don't think so. Here you can check which data providers are available in the self-hosted version: https://ghostfol.io/en/faq/self-hosting

**Comment by atiffarrukh** [comment]
You are right. Thanks for the quick response
